exercicios/para-casa/index.js

```diff
@@ -0,0 +1,92 @@
+  const express = require ("express")
+  const app = express()
+  const listaDeUsuarios = require("./model/usuarios.json")
+  const port = 3003
+  app.use(express.json())
+  
+
+  //- [X] Uma rota que atualiza todos os dados de cadastro de um usuário e se não for encontrado cria um novo na lista
+
+  app.put("/usuarios/:id",(req,res) => {
+
+   const idUsuario = req.params.id
+   const usuarioAlterado= req.body
+
+   const novaListaUsuario = listaDeUsuarios.map((usuario) => {
```

Comment:
Posso supor que essa lógica não está completamente implementada, certo? Nesse caso, quando você ainda estiver corrigindo o PR é interessante mudar o título dele pra `Draft:` ( significa que ainda não está pronto )

Outra descrição que pode colocar no PR para que dê a entender que não é a versão final é o `WIP` ( work in progress )

Reply:
Sim , profe, é pq estava com medo de ter problemas ai tentei logo. Vou fazer as modificações que sugeriu.